drivers/include/at.h

```diff
@@ -210,7 +263,9 @@ int at_dev_init(at_dev_t *dev, uart_t uart, uint32_t baudrate, char *buf, size_t
  * @param[in]   timeout timeout (in usec)
  *
  * @retval      0 when device answers "OK"
- * @retval     <0 otherwise
+ * @retval     -AT_ERR_EXTENDED if failed and there is a detailed error code in
+ *              @p dev->err_buf (i.e. DCE answered with "CMx ERROR")
```

Comment:
```suggestion
 *              @p dev->rp_buf (i.e. DCE answered with "CMx ERROR")
```
(same for the other occurrences)